src/chat/chat_panel.rs

```diff
@@ -466,7 +573,6 @@ impl Widget for ChatPanel {
                         } else {
                             item = list.item(cx, item_id, live_id!(UserChatLine)).unwrap();
                             chat_line_item = item.as_chat_line();
-                            chat_line_item.set_role("You");
```

Comment:
Changes in this PR would force us to thing a new name for the `set_role` function. Maybe `set_sender_name`?